src/ovirtapi/index.js

```diff
@@ -347,6 +349,13 @@ const OvirtApi = {
     assertLogin({ methodName: 'getStorageFiles' })
     return httpGet({ url: `${AppConfiguration.applicationContext}/api/storagedomains/${storageId}/files` })
   },
+  getIsoImages (): Promise<Object> {
```

Comment:
As I see, you added new line before functions, so, please, add it here too.

Reply:
I kinda used newlines to group a few related calls together visually.  `getIsoImages` groups with `getStorageFiles` so that's why there isn't a newline there.

Really I'd love to refactor this file a bunch.  Having `${AppConfiguration.applicationContext}` everywhere really kinda hurts my eyes.